marketplace/deployer_util/provision.py

```diff
@@ -53,6 +53,9 @@ def process(schema, values, deployer_image, deployer_entrypoint):
   app_name = get_name(schema, values)
   namespace = get_namespace(schema, values)
 
+  # Inject DEPLOYER_IMAGE property values if not already present.
+  values = inject_deployer_image_properties(schema, deployer_image, values)
```

Comment:
optional:
- remove the return value to make the API cleaner: it's modifying the passed in values directly
- Make `values` the first argument